### PR TITLE
Revised Subgraph Documents

### PR DESCRIPTION
**Pull request by PaulieB14**
Added entities, revised query examples/endpoints, and revised subgraph info

### CODE REVIEW

**Comment by PaulieB14** [comment]
@mendesfabio  - Hello - sorry forgot to revised summary.md.  Let me know if I need any changes.

File: references/Subgraph/Endpoints.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Ethereum Mainnet | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-v2)                   |
+| Polygon          | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-polygon-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-polygon-v2)   |
+| Arbitrum         | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-arbitrum-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-arbitrum-v2) |
+| Kovan            | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-kovan-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-kovan-v2)       |
+| Rinkeby          | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-rinkeby-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-rinkeby-v2)   |
```
Comment (mendesfabio) [inline references/Subgraph/Endpoints.md]: could you remove these links? we're deprecating Kovan and Rinkeby.

File: references/Subgraph/SubgraphData.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 1
+title: Subgraph Data Introduction
+---
+
+# Balancer Subgraph Introduction
+
+The Balancer Subgraph indexes data on the Balancer smart contracts with a GraphQL interface. It updates data in response to function calls and contract events to maintain data on the `Vault`, `Pools`, `AssetManagers` etc, to power front-end apps and integrations.
+
+Balancer has a GraphQL API Endpoint hosted by [The Graph](https://thegraph.com/docs/about/introduction#what-the-graph-is) called a subgraph for indexing and organizing data from the Sablier smart contracts.
```
Comment (mendesfabio) [inline references/Subgraph/SubgraphData.md]: This is quite confusing. I don't understand the reference to Sablier.

File: references/Subgraph/SubgraphData.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+# Balancer Subgraph Introduction
+
+The Balancer Subgraph indexes data on the Balancer smart contracts with a GraphQL interface. It updates data in response to function calls and contract events to maintain data on the `Vault`, `Pools`, `AssetManagers` etc, to power front-end apps and integrations.
+
+Balancer has a GraphQL API Endpoint hosted by [The Graph](https://thegraph.com/docs/about/introduction#what-the-graph-is) called a subgraph for indexing and organizing data from the Sablier smart contracts.
+
+This subgraph can be used to query Balancer data.
+
+Subgraph information is serviced by a decentralized group of server operators called Indexers.
```
Comment (mendesfabio) [inline references/Subgraph/SubgraphData.md]: This is only correct for Subgraph Studio. Balancer still uses the Hosted Service, which runs on centralized servers managed by Edge&Node.

File: references/Subgraph/SubgraphData.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+The schema of GraphQL elements available is defined in [`/schema.graphql` ](https://github.com/balancer-labs/balancer-subgraph-v2/blob/master/schema.graphql)
+
+## Ethereum Mainnet
+
+[Creating an API Key Video Tutorial](https://www.youtube.com/watch?v=UrfIpm-Vlgs)
+
+- [Explorer Page](https://thegraph.com/explorer/subgraph?id=CptFsHp6zar7kdfGYbVAqMeF1wNA1pJs6GaaJvPgeCfu&view=Overview)
+- Graphql Endpoint: https://gateway.thegraph.com/api/[api-key]/subgraphs/id/GAWNgiGrA9eRce5gha9tWc7q5DPvN3fs5rSJ6tEULFNM
+- [Code Repo](https://github.com/balancer-labs/)
```
Comment (mendesfabio) [inline references/Subgraph/SubgraphData.md]: This should be a link to the Subgraph repository, no?

**Review by mendesfabio — CHANGES_REQUESTED** [review]
thank you so much for this @PaulieB14! I left some comments for you to review. What I would love to see is the descriptions for the entities, would you be able to work on this? That would be super helpful as we could add the descriptions to the `schema.graphql` and make the developer's life easier. On a side note, we also have the [Subgraph for Gauges](https://github.com/balancer-labs/gauges-subgraph) that could have better docs.

File: references/Subgraph/SubgraphData.md (hunk 4 above)
Comment (PaulieB14) [reply references/Subgraph/SubgraphData.md]: @mendesfabio I made correction on this.

File: references/Subgraph/SubgraphData.md (hunk 3 above)
Comment (PaulieB14) [reply references/Subgraph/SubgraphData.md]: I will remove this sentence.

File: references/Subgraph/SubgraphData.md (hunk 2 above)
Comment (PaulieB14) [reply references/Subgraph/SubgraphData.md]: Sorry that was meant to say Balancer.

File: references/Subgraph/Endpoints.md (hunk 1 above)
Comment (PaulieB14) [reply references/Subgraph/Endpoints.md]: I deleted these out

**Comment by PaulieB14** [comment]
@mendesfabio I made changes.  Let me know if need further revisions.  I am newer to this so may take me some time to familiarize myself with balancer in order to provide descriptions.   I can work on it and do another PR if you want to put this through.  I can definitely help for your other subgraph as well.  I am working on descriptions now and will do a commit for them

**Comment by PaulieB14** [comment]
Is this the gauges subgraph you wanted docs on? There seems to be multiple on hosted?  https://thegraph.com/hosted-service/subgraph/mendesfabio/balancer-gauges-goerli

**Comment by PaulieB14** [comment]
@mendesfabio @markusbkoch @gerrrg  I finally finished descriptions.  Please let me know if we can finalize this PR!!  Thank you All!  If you help me finalize this, I can do a PR to add descriptions to your Schema if you'd like.   ¯\_(ツ)_/¯

**Comment by mikemcdonald** [comment]
I think I mostly got these changes moved over to the new docs repo at https://github.com/balancer-labs/docs. Thanks

**Pull request closed without merging**